### PR TITLE
BREAKING CHANGE: MUI migration v4 to v5

### PR DESCRIPTION
**Pull request by Anon-136**
## Why did you create this PR
- System props is much better in the new MUI v5 (full version)
- Fully use emotion

## What did you do
- Upgrade dependencies `@material-ui` -> `@mui` followed from [this guide](https://mui.com/guides/migration-v4/#migrate-from-jss)
- Remove storybook because it's been outdated
- Change all uses of `useStyles`, `makeStyles` or any jss related to emotion (CR11 Page) 
- Implement emotion cache for ssr followed from mui [Next.js reference](https://mui.com/guides/server-rendering/#reference-implementations)

## Demo
[https://dev.cugetreg.com](https://dev.cugetreg.com)

## Checklist
- [x] Deploy a demo
- [x] Check browsers compatibility
- [ ] Wrote coverage tests

### Other thing to tell us
Please check on every page and every component

### CODE REVIEW

**Review by saenyakorn — APPROVED** [review]
dev works = OK

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -39,42 +37,75 @@ export default class MyDocument extends Document {
...
-      })
-    }
-    return originalRenderPage({
-      enhanceApp: (App) => (props) => sheets.collect(<App {...props} />),
+  const cache = createEmotionCache()
+  const { extractCriticalToChunks } = createEmotionServer(cache)
+
+  ctx.renderPage = () =>
+    originalRenderPage({
+      enhanceApp: (App: any) => (props) => <App emotionCache={cache} {...props} />,
```
Comment (new5558) [inline src/pages/_document.tsx:48]: Eslint warning

**Review by new5558 — APPROVED** [review]
Lighthouse performance not change, test passed -> LGM

**Review by paphonb — APPROVED** [review]
Looks good!

**Pull request merged**